homepage/migrations/0003_course_test_data.py

```diff
@@ -0,0 +1,38 @@
+from django.db import migrations, transaction
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('homepage', '0001_initial'),
+    ]
+
+    def generate_data(apps, schema_editor):
+        from homepage.models import Course
+        from homepage.models import Prerequisites
+
+        test_data_course = [
+            (10231, 'UnFogging the Future', 1, 4, 'Currently Unavailable', 3.25, 1.5, 12, 12),
+            (10221, 'Grammatica in Arithmancy', 0, 3, 'Currently Unavailable', 4.25, 4.5, 10, 9),
+            (10111, 'Resonance in Runes and Signs', 1, 3, 'Currently Unavailable', 2.25, 2.0, 13, 13),
+            (12357, 'Numerology', 0, 5, 'Currently Unavailable', 3.5, 2.5, 7, 3),
+        ]
+
+        courses = [Course(course_id=t[0], name=t[1], mandatory=t[2],
+                          credit_points=t[3], syllabi=t[4], avg_rating=t[5],
+                          avg_load=t[6], num_of_raters=t[7], num_of_reviewers=t[8]) for t in test_data_course]
+
+        prerequisite_1 = Prerequisites(course_id=courses[3], req_course_id=courses[0], req_code=1)
+        prerequisite_2 = Prerequisites(course_id=courses[2], req_course_id=courses[1], req_code=0)
```

Comment:
maybe you can do the same as you did in lines # 14-23 like:
prerequisites = [
    (courses[3], courses[0], 1),
     (courses[2], courses[1], 0),
]
...

in this way you also can easily add/remove prerequisites